.github/workflows/licensing.yml

```diff
@@ -1,24 +0,0 @@
-# Copyright (C) 2017-2021  The SymbiFlow Authors.
```

Comment:
Why is the licensing check being removed?

Reply:
Looks like it is because it is now done by the `ci.yml`?

Reply:
Indeed, the licenses and python checks were executed twice